cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -508,6 +508,11 @@ func StartControllers(ctx ControllerContext, startSATokenController InitFunc, co
 			unsecuredMux.UnlistedHandle(basePath, http.StripPrefix(basePath, debugHandler))
 			unsecuredMux.UnlistedHandlePrefix(basePath+"/", http.StripPrefix(basePath, debugHandler))
 		}
+		if unsecuredMux != nil {
+			for _, healthzHandler := range healthzHandlers {
+				healthz.InstallPathHandler(unsecuredMux, fmt.Sprintf("/healthz/controllers/%s", controllerName), healthzHandler)
```

Comment:
Yeah, the prefix is good.

Reply:
InstallPathHandler has checks as variadic, so not sure there is any point in having this inside a for loop.